src/Commands/Admin.py

```diff
@@ -151,6 +154,52 @@ def brokenQueue(player: Member, roles: List[Role]) -> Embed:
     )
 
 
+async def forceReport(mentions: str, roles: List[Role], lbChannel: Channel, *arg) -> Embed:
+    if (Queue.isBotAdmin(roles)):
+
+        if (len(arg) > 0 and "<@!" in arg[0]):
```

Comment:
You removed the `len(arg) == 2` but didn't update `len(arg) > 0` to `len(arg) == 2`. Is it not actually needed here?

Reply:
I'm changing `len(arg) > 0` to `len(arg) == 2` in the next commit. Technically we don't need it but in the first `if` statement. It won't get further than the first one if there isn't at least two arguments to begin with. Instead of asking three times, we'll ask once at the beginning because the requirement is already satisfied thereafter.